File: thecode.js

var user1 = {
    name: 'Вася',
    age: 23,
    adress: 'Чебоксары'
};

var user2 = {
    name: 'Маша',
    age: 19,
    adress: 'Белгород'
};


function user_add_bonus(user, bonus_count) {
    user.bouns_count += bonus_count;
    user.bonus_level = Math.floor(user.bouns_count / 10000);
    if (user.bonus_level > 3) {
        user.bonus_level = 3;
    }
    console.log('Бонусный уровень покупателя ', user.name, ' : ', user.bonus_level)
}

File: thecode.py

class User:
    user_count = 0

    def __init__(self, name, age, adress):
        self.name = name
        self.age = age
        self.adress = adress
        User.user_count += 1

    def __del__(self):
        User.user_count -= 1


# user1 = User("ivan", 20, "addr1")
# user2 = User("huan", 30, "addr2")
# user3 = User("juan", 40, "addr3")

# print(User.user_count)
# # 3

# del user3
# print(User.user_count)
# # 3

user1 = User("ivan", 20, "addr1")
user2 = User("huan", 30, "addr2")
user3 = User("juan", 40, "addr3")


users = [user1, user2, user3]
print(len(users))
# 3

users.remove(user3)
print(len(users))
# 2


class User1:
    def __init__(self, name, age, adress):
        self.name = name
        self.age = age
        self.adress = adress

class User2:
    def __init__(self, name, age, adress):
        self.name = name
        self.age = age
        self.adress = adress

# ...

user1 = User1(...)
user2 = User2(...)

# ...
